samples/android/Preference/AddTagsPreference.cs

```diff
@@ -7,8 +7,11 @@
 using System.Collections.Generic;
 using System.Text;
 
+using Android.App;
```

Comment:
The code below here looks like a complete replacement, but it is actually very similar. Certain methods moved from `DialogPreference` to `PreferenceDialogFragmentCompat`, which seems to have confused the git diff engine.